src/traits.rs

```diff
@@ -1534,6 +1562,12 @@ pub trait ComponentExt: Debug {
     ///     component.refresh();
     /// }
     /// ```
+    ///
+    /// ## Linux
+    ///
+    /// - `temperature` is always read out form `hwmon`.
+    /// - `max` may kernel provided from `temp*-highest` or computed if
+    ///   `current_temperature` > `max`.
```

Comment:
I think there is no need to add both information here since we talk about them above.